test/cli/daemon.js

```diff
@@ -81,14 +81,22 @@ describe('daemon', () => {
         Gateway: '/ip4/127.0.0.1/tcp/0'
       }), '--json')
     }).then(() => {
-      return ipfs('daemon')
-    }).then((res) => {
-      expect(res).to.have.string('Daemon is ready')
-      done()
+      const proc = ipfs('daemon')
```

Comment:
Not sure what this change is, think @richardschneider did this. Is this really neccessary for this branch? Seems like a unrelated change.

Reply:
ping @richardschneider